include/databaseFunctions.php

```diff
@@ -1,34 +1,24 @@
 <?php
```

Comment:
Right now this works because you don't have much code, but as your code base grows, you'll want more than one file for database functions. I'd break it down by function. Like, I'd call this blog.php or something and then if, for example, you wanted to add a calendar to your site, you'd make calendar.php and if you added a system for users to sign up/log in, you could make user.php.